lib/ApplicationFeatures/V8SecurityFeature.cpp

```diff
@@ -278,8 +278,19 @@ void V8SecurityFeature::start() {
 
   _filesWhitelistRegex =
       std::regex(_filesWhitelist, std::regex::nosubs | std::regex::ECMAScript);
+
+}
+
+void V8SecurityFeature::dumpAccessLists() const {
+  LOG_TOPIC("2cafe", DEBUG, arangodb::Logger::SECURITY) << "files whitelisted by user:";
```

Comment:
can we reduce the number of messages here? It is not useful to have separate log lines for keys and values.
Instead, please do:
```
  LOG_TOPIC("2cafe", DEBUG, arangodb::Logger::SECURITY) << "files whitelisted by user: " << _filesWhitelist;
```
etc.